checks/fileparser/github_workflow_test.go

```diff
@@ -146,3 +146,111 @@ func TestGitHubWorkflowShell(t *testing.T) {
 		})
 	}
 }
+
+func TestIsWorkflowFile(t *testing.T) {
+	t.Parallel()
+	type args struct {
+		pathfn string
+	}
+	tests := []struct {
+		name string
+		args args
+		want bool
+	}{
+		{
+			name: "yaml",
+			args: args{
+				pathfn: "../testdata/github-workflow-shells-all-windows.yaml",
```

Comment:
can we use a `testdata/` in the current directory?

Reply:
The path is never used in the tests. It can also be a non-existent-path.